use-cases/GPU-passthrough-and-Kata.md

```diff
@@ -0,0 +1,276 @@
+
+   ```
+   $ ls /dev/dri
+     card0  renderD128
+   ```
+
+## GVT-g with Kata Containers
+
+For GVT-g, you need to append `i915.enable_gvt=1` in addition to `intel_iommu=on`
+to your host kernel command line. Make sure to reboot your host on
+making these changes.
+
+Follow  the steps below to pass an Intel Graphics device in GVT-d mode to a Kata Container:
+
+1. Find the BDF for GPU device:
+
+   ```
+   $ sudo lspci -nn -D | grep Graphics
+     0000:00:02.0 VGA compatible controller [0300]: Intel Corporation Broadwell-U Integrated Graphics [8086:1616] (rev 09)
+   ```
+
+   Run the above command to find out the BDF for the GPU device on host.
+   From the above output, PCI addres "0000:00:02.0" is assigned to the GPU device.
+   This BDF will be used later to unbind the GPU device from host.
+
+2. Choose the MDEV (Mediated Device) type for VGPU(Virtual GPU):
+
+   For background on mdev types, please follow this [kernel documentation](https://github.com/torvalds/linux/blob/master/Documentation/vfio-mediated-device.txt)
+
+   * List out the mdev types for the VGPU:
+
+     ```
+     $ BDF="0000:00:02.0"
+
+     $ ls /sys/devices/pci0000:00/$BDF/mdev_supported_types
+       i915-GVTg_V4_1  i915-GVTg_V4_2  i915-GVTg_V4_4  i915-GVTg_V4_8
+     ```
+
+   * Inspect the mdev types and choose one that fits your requirement:
+
+     ```
+     $ cd /sys/devices/pci0000:00/0000:00:02.0/mdev_supported_types/i915-GVTg_V4_8 && ls
+       available_instances  create  description  device_api  devices
+
+     $ cat description
+       low_gm_size: 64MB
+       high_gm_size: 384MB
+       fence: 4
+       resolution: 1024x768
+       weight: 2
+
+     $ cat available_instances
+       7
+     ```
+
+     The output of file `description` represents the GPU resources that will be assigned to the VGPU with specified MDEV type.<br/>
+     The output of file `available_instances` represents the remaining amount of VGPU can be created with specified MDEV type.
+
+3. Create a VGPU:
+
+   * Generate a uuid:
+
+     ```
+     $ # Generate a uuid
+     $ gpu_uuid=$(uuid)
+     ```
+
+   * Write the uuid to the `create` file under the chosen mdev type
+
+     ```
+     $ # Write the uuid to the `create` file under the chosen mdev type
+     $ echo $(gpu_uuid) | sudo tee /sys/devices/pci0000:00/0000:00:02.0/mdev_supported_types/i915-GVTg_V4/create
+     ```
+
+4. Find the IOMMU group for the VGPU:
+
+   ```
+   $ ls -la /sys/devices/pci0000:00/0000:00:02.0/mdev_supported_types/i915-GVTg_V4/devices/${gpu_uuid}/iommu_group
+     lrwxrwxrwx 1 root root 0 May 18 14:35 devices/bbc4aafe-5807-11e8-a43e-03533cceae7d/iommu_group -> ../../../../kernel/iommu_groups/0
+
+   $ ls -l /dev/vfio
+     total 0
+     crw-------   1 root root  241,   0 May 18 11:30 0
+     crw-rw-rw- 1 root root  10, 196 May 18 11:29 vfio
+   ```
+
+   From the above output, IOMMU group "0" is created.<br/>
+   The device node "/dev/vfio/0" can now be used in docker command line to pass
```

Comment:
Now you can use the device node "/dev/vfio/0" in the docker command line to pass